samples/features/security/always-encrypted-with-secure-enclaves/sample-application/.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
.git* files should not be checked into source control. Can you remoe these files?

Reply:
Done